### DISCUSSION TITLE
Add the Reflect and/or Inspectable traits to tilemap structs so we can use them nicely with egui_inspector

### DISCUSSION THREAD

**Discussion by dh7892** [StarArawn/bevy_ecs_tilemap#378] [Ideas]
As the title suggests.

I'm pretty new to Rust and very new to Bevy and even newer to becy_ecs_tilemap. But I've been playing around with it for a few days.

To help be work out what's what, I've been using bevy-inspector-egui. However, it won't automatically let me see details of the components that belong to the bevy_ecs_tilemap crate because they haven't got Reflect or Inspect implemented. 

It might be nice if you derived those traits for people to use with the inspector-egui, which I believe is pretty popular,

Thanks

**Comment by StarArawn** [comment]
Please add this as an issue. We should probably hide it under a feature flag.

**Comment by dh7892** [comment]
Done: https://github.com/StarArawn/bevy_ecs_tilemap/issues/412
